README.md

```diff
@@ -1,4 +1,4 @@
-# Rocket Academy Coding Bootcamp: Project 1: Frontend App
+# Project 1: Frontend App - Aime (WIP)
 
```

Comment:
It would be good to include instructions on how to install and run the app. Also a description of your project, as detailed as you like - but at least some kind of overview.